cfme/tests/infrastructure/test_vm_reconfigure.py

```diff
@@ -296,6 +296,38 @@ def test_vm_reconfig_resize_disk_hot(disk_type, disk_mode):
     pass
 
 
+@pytest.mark.manual
+@pytest.mark.tier(1)
+@pytest.mark.provider([VMwareProvider], override=True)
+@pytest.mark.parametrize('disk_type', ['thin', 'thick'])
+@pytest.mark.parametrize(
+    'disk_mode', ['persistent', 'independent_persistent', 'independent_nonpersistent'])
+def test_vm_reconfig_resize_disk_snapshot(disk_type, disk_mode):
```

Comment:
required:
You mark provider but not pass as arg... you will miss `privider` in test id.